### ISSUE TITLE
releases should be sent from one good, known email, e.g., Dasharo Release Team releases@dasharo.com, with the ability to reply to the whole release team. This issue can be resolved when we switch our process and newsletter configuration

### ISSUE THREAD

**Issue by macpijan** [Dasharo/dasharo-issues#567]

**Comment by macpijan** [comment]
We generally send from: `3mdeb Release Team <marketing@3mdeb.com>` for the last many of releases.

Last PC Engines SeaBIOS release used: `Dasharo Team <contact@dasharo.com>`

Our process currently do not specify concrete field.

@pietrushnic is the first one ok?

**Comment by pietrushnic** [comment]
@macpijan I would switch to the Dasharo/Zarhus Release Team. 3mdeb made sense when we were at an earlier stage; also, sending from marketing@3mdeb.com is a bad idea. Of course, it made sense for some types of newsletters but not for releases. contact@dasharo.com may also be a bad idea, and we should have an email where the real Dasharo Release Team resides. Who is in that team? Whovere can be a reviewer and approve MRs in the Dasharo Release Toolkit,

**Comment by miczyg1** [comment]
> We generally send from: `3mdeb Release Team <marketing@3mdeb.com>` for the last many of releases.
> 
> Last PC Engines SeaBIOS release used: `Dasharo Team <contact@dasharo.com>`

On the other hand, I always use `Dasharo Release Team <marketing@dasharo.com` , because that's what I have been taught: Dasharo releases are supposed to have Dasharo name in the email and entity releasing it.

**Comment by macpijan** [comment]
Yes, I probably should have written: `Dasharo Release Team <marketing@dasharo.com>`

Does this make sense then, to put it as a default one in the process?

**Comment by pietrushnic** [comment]
@macpijan I have doubts about marketing email use; it can easily fall into spam filters or be ignored. Also, the release notes we send are more than just marketing, especially for our customers for whom we fulfill our Terms of Service duties. Let's check what emails are used by others to get notifications for releases:

- Xous - Crowd Supply <no-reply@crowdsupply.com>
- mbedOS - info@mbed.com

**Comment by BeataZdunczyk** [comment]
we are using Dasharo Team <contact@dasharo.com>. Is this enough to close this issue, @macpijan ?
